src/components/common/NetworkSelector/index.tsx

```diff
@@ -90,20 +114,43 @@ const NetworkSelector = (props: { onChainSelect?: () => void }): ReactElement =>
         '& .MuiSelect-select': {
           py: 0,
         },
-        '& svg path': {
-          fill: ({ palette }) => palette.border.main,
-        },
       }}
     >
-      {configs.map((chain) => {
-        return (
-          <MenuItem key={chain.chainId} value={chain.chainId} disabled={isSocialLogin && !isSocialWalletEnabled(chain)}>
-            <Link href={getNetworkLink(chain.shortName)} onClick={props.onChainSelect} passHref>
-              <ChainIndicator chainId={chain.chainId} inline />
```

Comment:
Wouldn't it make more sense to include a logo into the ChainIndicator component?

Reply:
I started adding the changes to the `ChainIndicator` but it looks a bit out of place in the current safe list sidebar. Should we still push the changes since the safe list sidebar is getting reworked soon? Otherwise I could also only add the changes for inline chain indicators and leave the safe list out that way

Reply:
It could be an optional prop.

Reply:
I've updated the component and display the logo by default now. I've disabled it for the sidebar and safe list sidebar.

For the safe apps preview and share page I show the logos but increased the gap between each item slightly.